### PR TITLE
Update deploy.sh to deploy rbac

### PR DESCRIPTION
**Pull request by lpichler**
Changes:
- deploy script which is deploying also rbac
- readme file with mentioned changes

### How to run:

```
 ./deploy.sh rbac <path_to_local_copy_of_insights_rbac>
```

`path_to_local_copy_of_insights_rbac` is absolute path this [repository](https://github.com/RedHatInsights/insights-rbac)

Example:
```
./deploy.sh rbac /Users/liborpichler/Projects/insights-rbac
```

### Output

```
....bonfire output....

Route: XXXXX/api/authz/v1/relationships

user: XXXX
pass: XXXX

RBAC - status request consist creation of relations(image from PR https://github.com/RedHatInsights/insights-rbac/pull/1060)

curl -v -u xxxx:XXXX XXXX/api/rbac/v1/status/

Relations - Read(GET) - Sample CURL request

curl -v -u xxxx:XXXX XXXX 'XXXX/api/authz/v1/relationships?filter.objectType=group&filter.objectId=bob_club&filter.relation=member'

Relations - Write(POST) - Sample CURL request

curl -v -u xxxx:XXXX XXXX XXXXapi/authz/v1/relationships -d '{ touch: true, relationships: [{object: {type: group,id: bob_club},relation: member,subject: {object: {type: user,id: bob}}}]}'
```

### Links
https://issues.redhat.com/browse/RHCLOUD-31229

### CODE REVIEW

**Comment by Rajagopalan-Ranganathan** [comment]
@lpichler - has the deploy script been tested? Would be good to get the results. 
Also, the script should take an argument to deploy the RBAC rather than that being the default behavior. WDYT?

**Comment by lpichler** [comment]
> @lpichler - has the deploy script been tested? Would be good to get the results. Also, the script should take an argument to deploy the RBAC rather than that being the default behavior. WDYT?

yes it was and I will add everything what did you mentioned, thanks !

**Comment by lpichler** [comment]
cc @abaiken @coderbydesign

File: README.md [hunk 1]
```diff
@@ -95,3 +95,21 @@ The deploy script under the deploy folder, will deploy all the needed components
...
+
+```
+./deploy.sh rbac <path_to_local_copy_of_insights_rbac>
+```
+
+`path_to_local_copy_of_insights_rbac` is this [repository](https://github.com/RedHatInsights/insights-rbac)
+
+Example:
+```
+./deploy.sh rbac /Users/liborpichler/Projects/insights-rbac
```
Comment (Rajagopalan-Ranganathan) [inline README.md:110]: You cannot have your Kerberos ID here, can you please update it accordingly to something generic

File: deploy/deploy.sh [hunk 2]
```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
+
+RBAC_ARGUMENT="$1"
+RBAC_DIR="$2"
+if [ "$RBAC_ARGUMENT" == "rbac" ]; then
+
```
Comment (Rajagopalan-Ranganathan) [inline deploy/deploy.sh:6]: space/empty new line can be removed

File: deploy/deploy.sh [hunk 3]
```diff
@@ -67,7 +93,7 @@ echo "postgress is ready"
 oc create configmap spicedb-schema --from-file=schema.yaml -n $NAMESPACE
 
 #Deploy Relations service, spiceDB service
```
Comment (Rajagopalan-Ranganathan) [inline deploy/deploy.sh:95]: comment needs to be updated

**Review by Rajagopalan-Ranganathan — APPROVED** [review]

**Comment by Rajagopalan-Ranganathan** [comment]
LGTM - thanks a lot @lpichler for getting this done

**Pull request merged**